eg/Classes/Document.py

```diff
@@ -18,8 +18,8 @@
 
 import os
 import wx
-from tempfile import mkstemp
 from threading import Lock
+from tempfile import mkstemp
```

Comment:
Why is the import moved to another line? Does it make any difference? If not, i would like to have the change undone to make the diff shorter.